### ISSUE TITLE
How to open modal window through state param?

### ISSUE THREAD

**Issue by viryys** [peacechen/react-native-modal-selector#62]
can I create pull request with new props that will be open modal window?
example:
```
<ModalSelector
          data={attachTypes}
          cancelText="Cancel"
          animationType="fade"
          modalVisible="true"
          onChange={this.onPressYourMessage}>
          <View>
            <Icon name="ellipsis-h" />
          </View>
</ModalSelector>
```
if true modal window open, else close.
what you think about it?

**Comment by peacechen** [comment]
See #46
This has been implemented in e8d0fd2230799998f97984435d1be34069996fe3

**Comment by viryys** [comment]
oh, thank you, good work, really fast
